From 5e601ae8d0a15697206a5ff7a7a99326e9bbbcc0 Mon Sep 17 00:00:00 2001
From: Carsten Haitzler <raster@openmoko.org>
Date: Wed, 1 Oct 2008 17:30:23 +1000
Subject: [PATCH] list support for line dither -b ut don't use. the quality when using subtle
 gradients is just a little too low i think. but it's an option for about 10%
 speedup.

---
 conf/distro/include/sane-srcrevs.inc |    2 +-
 packages/efl1/evas_svn.bb            |   26 +++++++++++++-------------
 2 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/conf/distro/include/sane-srcrevs.inc b/conf/distro/include/sane-srcrevs.inc
index 3992cee..4a9bc7d 100644
--- a/conf/distro/include/sane-srcrevs.inc
+++ b/conf/distro/include/sane-srcrevs.inc
@@ -194,7 +194,7 @@ SRCREV_pn-zhone ?= "be3d72c6e5f5c0733df7064a616f8e39c1c12f2b"
 # Enlightenment Foundation Libraries
 # Caution: This is not alphabetically, but (roughly) dependency-sorted.
 # Please leave it like that.
-EFL_SRCREV ?= "36356"
+EFL_SRCREV ?= "36373"
 SRCREV_pn-edb-native ?= "${EFL_SRCREV}"
 SRCREV_pn-edb ?= "${EFL_SRCREV}"
 SRCREV_pn-eet-native ?= "${EFL_SRCREV}"
diff --git a/packages/efl1/evas_svn.bb b/packages/efl1/evas_svn.bb
index a94ae1f..5a78505 100644
--- a/packages/efl1/evas_svn.bb
+++ b/packages/efl1/evas_svn.bb
@@ -1,25 +1,24 @@
 require evas.inc
-PR = "r3"
+PR = "r8"
 
 EXTRA_OECONF = "\
 		--x-includes=${STAGING_INCDIR}/X11  \
 		--x-libraries=${STAGING_LIBDIR} \
-        --enable-evas-magic-debug \
-        \
-		--enable-fb					\
-		--disable-directfb			\
-		--disable-sdl				\
-		--enable-buffer				\
+	        --enable-evas-magic-debug 	\
+		--enable-fb			\
+		--disable-directfb		\
+		--disable-sdl			\
+		--enable-buffer			\
 		--disable-software-ddraw	\
 		--disable-software-qtopia	\
 		--enable-simple-x11		\
 		--enable-software-x11		\
 		--enable-software-16-x11	\
 		--disable-software-xcb		\
-		--disable-gl-x11			\
+		--disable-gl-x11		\
 		--enable-xrender-x11		\
 		--disable-xrender-xcb		\
-		--disable-glitz-x11			\
+		--disable-glitz-x11		\
 		--enable-image-loader-eet	\
 		--disable-image-loader-edb	\
 		--disable-image-loader-gif	\
@@ -28,13 +27,14 @@ EXTRA_OECONF = "\
 		--enable-image-loader-tiff	\
 		--enable-image-loader-xpm	\
 		--enable-image-loader-svg	\
-		--enable-cpu-c				\
-		--enable-fontconfig			\
+		--enable-cpu-c			\
+		--enable-fontconfig		\
 		--enable-font-loader-eet	\
 		--enable-scale-sample		\
 		--enable-scale-smooth		\
 		--enable-convert-yuv		\
 		--enable-small-dither-mask	\
+		--disable-line-dither-mask	\
 		--disable-no-dither-mask	\
 		--disable-convert-8-rgb-332	\
 		--disable-convert-8-rgb-666	\
@@ -49,7 +49,7 @@ EXTRA_OECONF = "\
 		--disable-convert-16-rgb-ipq	\
 		--enable-convert-16-rgb-rot-0	\
 		--enable-convert-16-rgb-rot-90	\
-		--disable-convert-16-rgb-rot-180 \
+		--disable-convert-16-rgb-rot-180\
 		--enable-convert-16-rgb-rot-270	\
 		--enable-convert-24-rgb-888	\
 		--enable-convert-24-bgr-888	\
@@ -59,5 +59,5 @@ EXTRA_OECONF = "\
 		--enable-convert-32-bgrx-8888	\
 		--enable-convert-32-rgb-rot-0	\
 		--enable-convert-32-rgb-rot-90	\
-		--disable-convert-32-rgb-rot-180 \
+		--disable-convert-32-rgb-rot-180\
 		--enable-convert-32-rgb-rot-270"
-- 
1.5.4.3

